packages/web3-core/src/web3_subscription_manager.ts

```diff
@@ -114,7 +141,11 @@ export class Web3SubscriptionManager<
 		this._subscriptions.delete(id);
 		return id;
 	}
-
+	/**
+	 * Will unsubscribe all subscriptions that fulfill the condition
+	 * @param condition : A function that access and `id` and a `subscription` and return `true` or `false`
```

Comment:
```suggestion
	 * @param condition - A function that access and `id` and a `subscription` and return `true` or `false`
```
Always use `-` after the parameter name, please. Actually, we got a lint warning for every missed dash. Please apply this to all other methods' params as well...